BPA/Ch.12/binmix.stan

```diff
@@ -0,0 +1,36 @@
+// Simplest binomial mixture model
+data {
+  int<lower=0> R;       // Number of sites
+  int<lower=0> T;       // Number of temporal replications
+  int<lower=0> y[R, T]; // Counts
+  int<lower=0> K;       // Upper bound of population size
+}
+
+transformed data {
+  int<lower=0> max_y[R];
+
+  for (i in 1:R)
+    max_y[i] <- max(y[i]);
+}
+
+parameters {
+  real<lower=0> lambda;    // Mean population size
+  real<lower=0,upper=1> p; // Detection probability
+}
+
+model {
+  // Priors
+  lambda ~ gamma(0.005, 0.005);
```

Comment:
The population size is unknown, but it should be not so large; an alternative prior proposed in the book is uniform(0, 10). So that now I use a Cauchy prior on the parameter.